### PR TITLE
fix(2152): Pass in pipelineId and prNum to Sonar coverage [2]

### PR DESCRIPTION
**Pull request by tkyi**
## Context

Need to pass in pipelineId and prNum for new Sonar coverage calls.

## Objective

This PR passes in pipelineId and prNum to get Sonar coverage info.

## References

Related to https://github.com/screwdriver-cd/screwdriver/issues/2152

## License

I confirm that this contribution is made under the terms of the license found in the root directory of this repository's source tree and that I have the authority necessary to make this contribution on behalf of its copyright owner.

### CODE REVIEW

File: app/components/pipeline-list-view/component.js [hunk 1]
```diff
@@ -128,7 +131,9 @@ export default Component.extend({
           jobId,
           buildId,
           startTime: latestBuild.startTime,
-          endTime: latestBuild.endTime
+          endTime: latestBuild.endTime,
+          pipelineId: latestBuild.pipelineId,
+          prNum: prNumMatch ? prNumMatch[1] : null
```
Comment (pritamstyz4ever) [inline app/components/pipeline-list-view/component.js:136]: should we have a length check here as we are getting the second index?

Comment (tkyi) [reply app/components/pipeline-list-view/component.js]: updated

File: app/components/pipeline-list-view/component.js [hunk 2]
```diff
@@ -128,7 +131,9 @@ export default Component.extend({
           jobId,
           buildId,
           startTime: latestBuild.startTime,
-          endTime: latestBuild.endTime
+          endTime: latestBuild.endTime,
+          pipelineId: latestBuild.pipelineId,
+          prNum: prNumMatch && prNumMatch.length > 1 ? prNumMatch[1] : null
```
Comment (adong) [inline app/components/pipeline-list-view/component.js:136]: Just want to point out some cool features.

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Optional_chaining


```
prNumMatch?.[1] ?? null
```

Comment (jithine) [reply app/components/pipeline-list-view/component.js:136]: Cool feature. Reduces boiler plate a lot. For ember it doesn't matter, it always uses babel.  But we should use it in api and other components after upgdating to node:12

**Pull request merged**